src/oas-parsing/schema/oas-schema.ts

```diff
@@ -65,6 +65,13 @@ class OASSchema {
       options = { server, ...options };
     }
 
+    if (
+      options.parameters?.Accept !== undefined &&
+      options.parameters?.Accept.length > 0
+    ) {
```

Comment:
Should we also check if this parameter is a header parameter? Otherwise the `Accept` parameter is maybe ok?

Reply:
Good idea, added that check with latest commit.